### PR TITLE
Update Coverage Checkers to use either varname or product name

### PR DESCRIPTION
**Pull request by evrose54**
# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#227

# Summary
These changes reflect the need to simplify naming conventions in the coverage_checkers modules. Per issue #227, these names were ambiguous and tangled, and the new changes create an easier, simpler way to name the outputted coverage checkers. The files changed are listed below.

  - geoips/geoips/plugins/modules/coverage_checkers/center_radius_rgba.py
  - geoips/geoips/plugins/modules/coverage_checkers/center_radius.py
  - geoips/geoips/plugins/modules/coverage_checkers/masked_arrays.py
  - geoips/geoips/plugins/modules/coverage_checkers/numpy_arrays_nan.py
  - geoips/geoips/plugins/modules/coverage_checkers/rgba.py
  - geoips/geoips/plugins/modules/coverage_checkers/windbarbs.py
  - geoips/geoips/interfaces/module_based/coverage_checkers.py

### CODE REVIEW

**Comment by mindyls** [comment]
@evrose54 Have you run $GEOIPS_PACKAGES_DIR/geoips/tests/integration_tests/full_test.sh with these changes? If so, can you paste the output here?  If additional PRs are required (ie, on data_fusion repo, etc), please include links to all other required PRs as well.

There should be a complete 0 return for all updates - and if these changes aren't tested, please add a test to ensure these updates are fully tested prior to finalizing PR.

**Review by mindyls — CHANGES_REQUESTED** [review]
Please run $GEOIPS/tests/integration_tests/full_test.sh, and include the full 0 return in the PR contents.  Also link to any other PRs that were required to get the 0 return.

**Review by jsolbrig — APPROVED** [review]

**Review by mindyls — APPROVED** [review]
I'm good merging this, and I'll test after merge.

File: geoips/plugins/modules/coverage_checkers/center_radius.py [hunk 1]
```diff
@@ -110,21 +108,13 @@ def call(
...
     """
     varname_for_covg = variable_name
-    if variable_name not in xarray_obj.variables.keys() and alt_varname is not None:
+    if variable_name not in xarray_obj.variables.keys():
         LOG.info(
-            '    UPDATING variable "%s" does not exist, using alternate "%s"',
+            '    UPDATING variable "%s" does not exist, using product name "%s"',
             variable_name,
-            alt_varname,
+            name,
```
Comment (mindyls) [inline geoips/plugins/modules/coverage_checkers/center_radius.py:115]: Looking at this closer, I don't think this is what we want - it is using the name of the coverage checker plugin if the variable name does not exist in the xarray object.  I don't think that makes sense.  I will think about how this works, and expect an updated implementation when 1.11.1 is released.

**Pull request merged**